core/test/lib/asset-saver-test.js

```diff
@@ -33,14 +33,14 @@ describe('asset-saver helper', () => {
   describe('saves files', function() {
     const tmpDir = `${LH_ROOT}/.tmp/asset-saver-test`;
 
-    before(() => {
+    before(async () => {
       fs.mkdirSync(tmpDir, {recursive: true});
       const artifacts = {
         DevtoolsLog: [{message: 'first'}, {message: 'second'}],
         Trace: {traceEvents},
       };
 
-      return assetSaver.saveAssets(artifacts, dbwResults.audits, `${tmpDir}/the_file`);
+      await assetSaver.saveAssets(artifacts, dbwResults.audits, `${tmpDir}/the_file`);
```

Comment:
Ope, previous attempt had multiple calls to `saveAssets`, I'll change it back if someone wants